packages/rancher-cis-benchmark/charts/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: v1.0.1
+appVersion: v1.0.2
 description: The cis-operator enables running CIS benchmark security scans on a kubernetes
   cluster
 name: rancher-cis-benchmark
-version: 1.0.101
+version: 1.0.201
```

Comment:
not sure but I think this should be 1.0.200? Rest LGTM

Reply:
yup, done